pkg/crc/api/events/status.go

```diff
@@ -0,0 +1,72 @@
+package events
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/crc-org/crc/pkg/crc/logging"
+	crcMachine "github.com/crc-org/crc/pkg/crc/machine"
+	"github.com/r3labs/sse/v2"
+)
+
+type genData func() (interface{}, error)
+
+// TickListener a data generator for an event stream. It will fetch data at regular intervals, and
+// send it to all clients connected to the endpoint.
+type TickListener struct {
+	done       chan bool
+	generator  genData
+	tickPeriod time.Duration
+}
+
+func newStatusStream(server *EventServer) EventStream {
+	return newStream(NewStatusListener(server.machine), newEventPublisher(STATUS, server.sseServer))
+}
+
+func NewStatusListener(machine crcMachine.Client) EventProducer {
+	getStatus := func() (interface{}, error) {
+		return machine.GetClusterLoad()
```

Comment:
this'l only provide the memory, disk related info and not the openshift/microshift status right?

Reply:
cluster state would also need to interact over ssh, so this is easy to change. but seems you are right.

Reply:
perhaps related to https://github.com/crc-org/crc/pull/3700#issuecomment-1581888604 as that was `watch` logic?

Reply:
This is therefore existing behaviour and needs to be fixed in a follow-up issue: #3744